exporter/opentelemetry-exporter-prometheus-remote-write/README.rst

```diff
@@ -1,27 +1,248 @@
-OpenTelemetry Prometheus Remote Write Exporter
-==============================================
+OpenTelemetry Python SDK Prometheus Remote Write Exporter
+=========================================================
 
-This library allows exporting metric data to `Prometheus Remote Write Integrated Backends
-<https://prometheus.io/docs/operating/integrations/>`_. Latest `types.proto
-<https://github.com/prometheus/prometheus/blob/master/prompb/types.proto>` and `remote.proto
-<https://github.com/prometheus/prometheus/blob/master/prompb/remote.proto>` Protocol Buffers 
-used to create WriteRequest objects were taken from Prometheus repository. Development is 
-currently in progress.
+This package contains an exporter to send `OTLP`_ metrics from the
+Python SDK directly to a Prometheus Remote Write integrated backend
+(such as Cortex or Thanos) without having to run an instance of the
+Prometheus server. The image below shows the two Prometheus exporters in
+the OpenTelemetry Python SDK.
+
+Pipeline 1 illustrates the setup required for a Prometheus "pull"
+exporter.
+
+Pipeline 2 illustrates the setup required for the Prometheus Remote
+Write exporter.
+
+|Prometheus SDK pipelines|
+
+The Prometheus Remote Write Exporter is a "push" based exporter and only
```

Comment:
So this diagram and explanations are great. It seems to be describing how the metrics pipeline works in general, and not specific to this exporter so I'm wondering if we should move this somewhere else?

Reply:
do you have any places in mind? I thought it might be useful for people to know what the collection pipeline looks like but if there is no good place to put this I can just remove the section

Reply:
We can leave this for now.